ansible/eda-server
# [AAP-10890] Activation Instance schema bug

### Jira Ticket: [AAP-10890](https://issues.redhat.com/browse/AAP-10890) ###

### Purpose: ###
- The activation instance model has the foreign key for activation as a nullable field. We would like this field to cascade on delete and not allow for activation instances for a none existent activation. 

### Testing Instructions: ### 
1. load up project data
2. Create an activation: `[POST] http://127.0.0.1:8000/api/eda/v1/activations/`
3. Check to see if activation instances were created for the activation `[GET] http://127.0.0.1:8000/api/eda/v1/activation-instances/`
4. Delete the activation: `[DELETE] http://127.0.0.1:8000/api/eda/v1/activations/{activation_id}`
5. Ensure all activation instances related to the activation also delete.
